client/app/components/packages/rwcds-form/proposed-project-actions-editor.hbs

```diff
@@ -0,0 +1,48 @@
+{{#let @form as |Form|}}
+  <Ui::Question class="fieldset relative scale-fade-in" as |Q|>
+    <Q.legend>
+      <strong>
+        <span class="label">
+          {{~@landUseAction.dcpZoningresolutiontype~}}
+          {{#if (lookup-action-type @landUseAction.dcpZoningresolutiontype)}}
+            <LabsUi::IconTooltip
+              @tip={{lookup-action-type @landUseAction.dcpZoningresolutiontype}}
+              @icon="info-circle"
+              @side="bottom"
+              @transform="shrink-4 up-2"
+              @fixedWidth={{true}}
+            />
+          {{/if}}
+        </span>
+        {{#if (in-array this.actionsWithSectionNumber @landUseAction.dcpZoningresolutiontype)}}
+          {{@landUseAction.dcpZrsectionnumber}}
+        {{/if}}
+        {{#if (and (eq @landUseAction.dcpZoningresolutiontype 'ZR') (eq @landUseAction.dcpZrsectionnumber 'Appendix F'))}}
+          {{@landUseAction.dcpZrsectionnumber}}
+        {{/if}}
```

Comment:
This might work better as a computed (just so it's named explicitly as `this.actionTitle`)